Orange/widgets/data/owimpute.py

```diff
@@ -220,192 +110,170 @@ def __init__(self):
         )
         self.selection = self.varview.selectionModel()
 
-        box.layout().addWidget(self.varview)
-
-        method_layout = layout(Qt.Vertical, margins=0)
-        box.layout().addLayout(method_layout)
-
-        methodbox = group_box(layout=layout(Qt.Vertical))
+        horizontal_layout.addWidget(self.varview)
 
-        bgroup = QButtonGroup()
-        for i, m in enumerate(self.METHODS):
-            b = radio_button(m.name, group=bgroup, group_id=i)
-            methodbox.layout().addWidget(b)
+        method_layout = QtGui.QVBoxLayout()
+        horizontal_layout.addLayout(method_layout)
 
-        assert self.METHODS[-1].short == "value"
+        button_group = QtGui.QButtonGroup()
+        for i, method in enumerate(self.METHODS):
+            button = QtGui.QRadioButton(text=method.name)
+            button_group.addButton(button, i)
+            method_layout.addWidget(button)
 
-        self.value_stack = value_stack = QStackedLayout()
-        self.value_combo = QComboBox(
+        self.value_combo = QtGui.QComboBox(
             minimumContentsLength=8,
-            sizeAdjustPolicy=QComboBox.AdjustToMinimumContentsLength,
+            sizeAdjustPolicy=QtGui.QComboBox.AdjustToMinimumContentsLength,
```

Comment:
Forward compatible, it's better to import and refer to `QObjects` directly. In Qt 5, for example, `QComboBox` is in `QtWidgets` namespace (module).